content/applications/finance/payment_providers/asiapay.rst

```diff
@@ -2,23 +2,30 @@
 AsiaPay
 =======
 
-`AsiaPay <https://www.asiapay.com/>`_ is an online payments provider established in Hong Kong and
+`AsiaPay <https://www.asiapay.com/>`_ is an online payments provider established in Hong Kong,
 covering several Asian countries and payment methods.
 
 .. _payment_providers/asiapay/configure_dashboard:
 
 Configuration on AsiaPay Dashboard
 ==================================
 
-#. Log into `AsiaPay Dashboard <https://www.paydollar.com/b2c2/eng/merchant/index.jsp>`_ and go to
-   :menuselection:`Profile --> Account Information`. Copy the values of the :guilabel:`Currency` and
-   :guilabel:`Secure Hash` fields and save them for later.
+#. Log into AsiaPay Dashboard according to the account provided by AsiaPay.
+
+   - `PayDollar <https://www.paydollar.com/b2c2/eng/merchant/index.jsp>`_: For markets in HK,
+     CN, MO, TW, SG, MY, IN, VN, NZ and AU;
+   - `PesoPay <https://www.pesopay.com/b2c2/eng/merchant/index.jsp>`_: For market in PH;
+   - `SiamPay <https://www.siampay.com/b2c2/eng/merchant/index.jsp>`_: For market in TH;
+   - `BimoPay <https://www.bimopay.com/b2c2/eng/merchant/index.jsp>`_: For market in ID.
+
+#. | Go to :menuselection:`Profile --> Account Information`. Copy the values of the
+     :guilabel:`Currency` and :guilabel:`Secure Hash` fields and save them for later;
```

Comment:
This single line doesn't need the `|` because there is no line break to cancel.
```suggestion
#. Go to :menuselection:`Profile --> Account Information`. Copy the values of the
   :guilabel:`Currency` and :guilabel:`Secure Hash` fields and save them for later;
```

Reply:
Updated